loculus_values/environment_specific_values/staging.yaml

```diff
@@ -64,4 +64,11 @@ seqSets:
   crossRef: null
 sequenceFlagging:
   github:
-    issueTemplate: sequence-metadata-issue.md
\ No newline at end of file
+    issueTemplate: sequence-metadata-issue.md
+defaultResources:
+  requests:
+    memory: "200Mi"
+    cpu: "20m"
+  limits:
+    memory: "1Gi"
+    cpu: "20m"
```

Comment:
What's the reason you want to specify different resources for staging specifically? 

What's the logic behind:
- Slightly higher cpu requests (20 vs 10)
- But much lower cpu limits (20 vs 1000)
vs what's in values.yaml (and hence the setting for all non-prod namespaces unless they overwrite differently - none of them do except for prod)

Reply:
It just felt conservative to keep staging exactly as it was before my PR - which is what this does because previously it inherited these from the base Loculus `values.yaml` but I don't feel strongly - would also be happy to remove this special config!